releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,21 @@
 # Microsoft.FeatureManagement.AspNetCore
 [Source code ][source_code_web] | [Package (NuGet)][package_web] | [Samples][samples_web] | [Product documentation][docs]
 
+## 3.1.0 - November 23, 2023
+
+### Changes
```

Comment:
Bug fixes are changes too. Should this be "Enhancement"? I know this might be copied from the previous one. Can we fix that too?